app/controllers/admin_controller.rb

```diff
@@ -1,16 +1,12 @@
 class AdminController < ApplicationController
+  respond_to :html
   def index
     authorize! :manage, :all
-    respond_to do |format|
-      format.html # index.html.haml
-    end
   end
 
   def newsletter
     authorize! :manage, :all
     @members = Member.confirmed.wants_newsletter.all
-    respond_to do |format|
-      format.html # index.html.haml
-    end
+    respond_with @member
```

Comment:
Looks like a bug we missed. Better fix (and add regression tests) or newletter will break